### ISSUE TITLE
Problem with line breaks in the cell

### ISSUE THREAD

**Issue by kitar** [kzykhys/PHPCsvParser#5]
Hi,

PHPCsvParser is so helpful for me, but there's some little problem with line breaks in the cell.

Here's some csv file to test.

``` csv
hoge,"foo
bar",fuga
```

When I parse this file using CsvParser::fromFile(), It'll be fine. $result[0][1] contains line break.

``` php
$parser = \KzykHys\CsvParser\CsvParser::fromFile('./test.csv');
$result = $parser->parse();
```

But when I parse this file using CsvParser::fromString(), $result[0][1] doesn't contains line break.

``` php
$string = file_get_contents("./test.csv");
$parser = \KzykHys\CsvParser\CsvParser::fromString($string);
$result = $parser->parse();
```

Thanks for your help.

**Comment by kzykhys** [comment]
@kitar Thank you for reporting! I will release 1.4.1 after more tests in a few days.
If you need to fix this issue now, you can try `dev-master` for now.

``` json
{
    "require": {
        "kzykhys/php-csv-parser": "dev-master@dev"
    }
}
```

**Comment by kitar** [comment]
@kzykhys Thank you for your quick support! At this moment, I checked that this problem fixed at 13b1e79.

**Comment by kzykhys** [comment]
Just released 1.4.1

**Comment by kitar** [comment]
@kzykhys Thank you so much! I'll use it in my project.
